ETL_pipeline/load_flood.py

```diff
@@ -0,0 +1,146 @@
+"""Load flood warnings into the database"""
+
+from os import environ as ENV
+
+import psycopg2
+import psycopg2.extras
```

Comment:
Might have missed something but is this used in this file?

Reply:
i forgot to include cursorfactory